attributes/repo_hosting.rb

```diff
@@ -0,0 +1,19 @@
+# Default to previous behaviour of not hosting repos in phabricator
+# NB this setting has no effect on Ubuntu 12.04 as its version of OpenSSH
+# is too old to work.
+default['phabricator']['repo_hosting_enabled'] = false
+
+# Port that VCS SSH will listen on
+default['phabricator']['ssh_vcs_port'] = '617'
+
+# User for Source code hosting
+default['phabricator']['vcsuser'] = 'git'
```

Comment:
I suggest that the VCS functions are namespaced into e.g. `['phabricator']['vcs_hosting']` (or, preferably, something a little more elegant)